app/controllers/admin/admin_controller.rb

```diff
@@ -1,3 +1,19 @@
 class Admin::AdminController < ApplicationController
   layout "admin"
+
+  def admin_title(model=nil, keys = [])
+    return t('.index_title') unless model.present?
```

Comment:
this refers to any of the `index_title`s I added in the [`en.yml`][0] file.

this is using the "[lazy lookup][1]" dot notation from the Rails i18n feature

[0]:https://github.com/crimethinc/website/pull/338/files#diff-37c9b835927e3c363588f172de9bb6faR149
[1]:http://guides.rubyonrails.org/i18n.html#lazy-lookup